JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -35,6 +35,9 @@
         createLazyMethod("track" + method.pop());
     }
 
+    createLazyMethod("setAuthUserContext");
```

Comment:
Why do we need to change a snippet and we do not change the AppInsights object itself in the actual script and do not add those methods there? It will be more logical to add those methods into the actual AppInsights object so people with the old snippet can still use those method, when new customers may benefit of earlier loading of them.